src/content/en/2022/seo.md

```diff
@@ -1,22 +1,754 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: SEO
-#TODO - Review and update chapter description
 description: SEO chapter of the 2022 Web Almanac covering crawlability, indexability, page experience, on-page SEO, links, AMP, internationalization, and more.
 authors: [SophieBrannon, itamarblauer]
-reviewers: [mobeenali97, dwsmart, patrickstox, johnmurch, TusharPol]
-analysts: [derekperkins, csliva, jroakes]
+reviewers: [patrickstox, TusharPol, mobeenali97, dwsmart, johnmurch]
+analysts: [csliva, jroakes, derekperkins]
 editors: [MichaelLewittes]
 translators: []
 results: https://docs.google.com/spreadsheets/d/1qBQWxNKIAVJNOFwGlslT7AW0VAoK85Mf3nFvtw0QjVU/
-featured_quote: TODO
-featured_stat_1: TODO
-featured_stat_label_1: TODO
-featured_stat_2: TODO
-featured_stat_label_2: TODO
-featured_stat_3: TODO
-featured_stat_label_3: TODO
-unedited: true
+SophieBrannon_bio: TODO
+itamarblauer_bio: TODO
+featured_quote: The implementation of structured data in the HTML of a page has continually increased. In 2021, 41.8% of desktop pages and 42.5% of mobile pages used structured data. In 2022, it’s risen to 44% of desktop pages and 45.1% of mobile pages that have structured data within their HTML.
+featured_stat_1: 84.75%
+featured_stat_label_1: Sites adopting HTTPS
+featured_stat_2: 66%
+featured_stat_label_2: Websites that have implemented an H1 tag
+featured_stat_3: 20%
+featured_stat_label_3: Images using lazy loading image properties
 ---
 
-## TODO
+## Introduction
+
+Search engine optimization (SEO) is a digital technique used to improve a website or page's visibility so that it organically ranks higher in search  engine results. It often combines technical configuration, content creation, and link acquisition, with the goal of improving relevance for a searcher’s query and intent. SEO has continued to grow in popularity and become one of the most popular digital marketing channels.
+
+{{ figure_markup(
+  image="seo-term-trends.png",
```

Comment:
Who's providing this image?

Reply:
The link to the image is here - 

https://trends.google.com/trends/explore?date=all&geo=US&q=%2Fm%2F019qb_,%2Fm%2F03_hgq,social%20media%20marketing,%2Fg%2F1z2v7wknj,email%20marketing